protocol/messenger_communities.go

```diff
@@ -558,8 +558,15 @@ func (m *Messenger) CuratedCommunities() (*communities.KnownCommunitiesResponse,
 	if err != nil {
 		return nil, err
 	}
+
+	// TODO: The curated communities smart contract should also store the community shard cluster and index
+
 	for _, c := range featuredCommunities {
-		response.ContractFeaturedCommunities = append(response.ContractFeaturedCommunities, types.HexBytes(c).String())
+		response.ContractFeaturedCommunities = append(response.ContractFeaturedCommunities, communities.CommunityShard{
+			CommunityID: types.HexBytes(c).String(),
+			// Cluster:     c.Cluster, // TODO: obtain this value
+			// Index:       c.Index, // TODO: obtain this value
+		})
```

Comment:
The curated communities smart contract must be modified to also store the shard cluster and index for a community